client/main.lua

```diff
@@ -33,7 +33,19 @@ local DisableControlAction = DisableControlAction
 local DisablePlayerFiring = DisablePlayerFiring
 local options = {}
 local currentTarget = {}
+local activated = false
 
+function getTargetData()
+    return currentTarget
+end
+
+function getOptionsData()
+    return options
+end
+
+function getActivated()
+    return activated
+end
```

Comment:
I _just_ got rid of a bunch of global functions and now you want to add a bunch of dumb ones. Also - _activated_???????????